engine/src/main/java/org/pentaho/di/www/GetJobImageServlet.java

```diff
@@ -185,6 +194,27 @@ public void doGet( HttpServletRequest request, HttpServletResponse response ) th
     }
   }
 
+  private void buildDuplicateResponse( HttpServletResponse response, String jobName, boolean useXML ) {
```

Comment:
Please write a unit test for the method.

Reply:
Made several unit tests for GetJobImageServlet